### DISCUSSION TITLE
View edited file in browser

### DISCUSSION THREAD

**Discussion by ghost** [coder/code-server#2115] [Q&A] [answered]
Hey community,

I'm a spare-time web-developer and need to view the "result" of my changes in a few websites instantly. Does someone know how I can view a website which I edited in the code-server in my browser??

Your help would mean a lot to me.

thx

**Comment by code-asher** [comment, marked as answer]
code-server has a built in proxy you could use. For example if your web server is on port 8081 you could go to `code-server.tld/proxy/8081`. More details here: https://github.com/cdr/code-server/blob/v3.5.0/doc/FAQ.md#how-do-i-securely-access-web-services
